src/main/java/Main.java

```diff
@@ -283,4 +283,47 @@ private static String validateEmailInput() {
         } while (!matcher.matches());
         return email;
     }
+
```

Comment:
SSS, TIN, and Pagibig are supposed to accept character as well. Kindly fix this.